### PR TITLE
added units explanation

### PR DESCRIPTION
**Pull request by missxa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,3 +93,7 @@ muscle: 1
 action: 'move to'
 setpoint: 10.0" 
 ```
+### Units
+- Position controller ('move to' command) expects the setpoint in radians 
+- actuarorPosition and tendonDisplacement are displayed in encoder ticks
+- 1 rad ~ 18000 ticks
```
Comment (compiaffe) [inline README.md]: How do you get to this number?

It is depended on 
- the encoder (512 encoder tics/rotation in our case)
- the gear box (53:1) - see https://roboy.open-aligni.com/part/show/359?revision_id=316#
-----------
512*53 = 27136 tics per rotation of the tendon reel

27136 tics/rotation = 27136tics/2*pi RAD = 4318.955912781

File: README.md [hunk 2]
```diff
@@ -93,3 +93,7 @@ muscle: 1
 action: 'move to'
 setpoint: 10.0" 
 ```
+### Units
+- Position controller ('move to' command) expects the setpoint in radians 
+- actuarorPosition and tendonDisplacement are displayed in encoder ticks
```
Comment (compiaffe) [inline README.md]: actuatorPosition (typo)

**Review by compiaffe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (missxa) [reply README.md]: from the yaml file: rad_per_encoder_count: 0.00005788606746738269
ticks per rad = 1/ 0.00005788606746738269 ~ 18000

Comment (missxa) [reply README.md]: no?
i tested when controlling, seems to be true

Comment (compiaffe) [reply README.md]: hmm, are you sure?
encoder and gear box values are certainly correct.

How many tics do you measure when doing a full rotation (visually verified)?

Comment (compiaffe) [reply README.md]: Ahh, found what it was deep in the back of my mind:

- Gear box is correct at 53:1
- Encoder tics is also correct at 512 tics per rotation, BUT being a quadrature encoder a tic is actually 4 tics/pulses:

512*53*4=108544 tics per rotation 

108544/(2+3.1415) = 17275.8236 or ~18000 tics per rad

**Review by compiaffe — APPROVED** [review]

**Pull request merged**